From 9b9166dfa13d5071e9310d84a2ebeec89755dc6f Mon Sep 17 00:00:00 2001
From: rikardo <rikard.olander@hostmobility.com>
Date: Mon, 16 Aug 2021 14:15:42 +0200
Subject: [PATCH 7/7] Remove dma on spi (HC Ignore dma for now, use pio) This
 was meant to be used with mcp2515 (can controller) and was merge in with EG25
 driver support by misstake. Maybe we should remove this patch after test of
 co cpu without this patch.

---
 drivers/spi/spi-imx.c | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/drivers/spi/spi-imx.c b/drivers/spi/spi-imx.c
index 0ba0a1d4df68..b6a3ed4183fb 100644
--- a/drivers/spi/spi-imx.c
+++ b/drivers/spi/spi-imx.c
@@ -1702,15 +1702,8 @@ static int spi_imx_probe(struct platform_device *pdev)
 	 * Only validated on i.mx35 and i.mx6 now, can remove the constraint
 	 * if validated on other chips.
 	 */
-	if (spi_imx->devtype_data->has_dmamode) {
-		ret = spi_imx_sdma_init(&pdev->dev, spi_imx, master);
-		if (ret == -EPROBE_DEFER)
-			goto out_clk_put;
-
-		if (ret < 0)
-			dev_err(&pdev->dev, "dma setup error %d, use pio\n",
-				ret);
-	}
+
+	dev_err(&pdev->dev, "HC Ignore dma for now, use pio\n");
 
 	spi_imx->devtype_data->reset(spi_imx);
 
-- 
2.17.1

